frontend/src/app/components/MainInstallButton.js

```diff
@@ -49,11 +56,8 @@ export default class MainInstallButton extends React.Component {
       </div>
     );
   }
-
-  install() {
-    const { eventCategory, hasAddon } = this.props;
-    if (hasAddon) { return; }
-    this.setState({ isInstalling: true });
-    installAddon(eventCategory);
-  }
 }
+
+MainInstallButton.contextTypes = {
```

Comment:
I tried to keep the surface area of this PR small because of its urgency, but yep, that's exactly what the promised land looks like.